magento/pwa-studio
# [Doc] project structure topic

## Description

This PR revises the project structure and routing tutorial from @rossmc .
Content about the project structure has been expanded on and the routing parts have been moved to the existing routing topic in peregrine. The existing routing topic has also been updated to reflect the current implementation details.

## Related Issue

Closes PWA-300

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory
1. Run the linter on the modified pages:
    ```sh
    npm run lint src/peregrine/routing/index.md
    npm run lint src/tutorials/pwa-studio-fundamentals/
    ```
1. Verify no linting errors.
1. Build the preview site: `npm run develop`
1. Verify site builds with no errors
1. Navigate to the _Tutorials_ section
1. Verify _Project structure_ link appears on the left nav

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
